domain/tutoringModel/utils.py

```diff
@@ -113,3 +113,33 @@ def random_generator(num, size, type_):
         return rng.choice([True, False], size)
     else:
         raise err.WrongParameterValueError()
+
+
+# Check if learning style is correctly formatted and
```

Comment:
can be written as function docstring to give context when hovering

Reply:
Agree and resolved